iogt/static/css/modules/_m-pagination.scss

```diff
@@ -9,5 +9,5 @@
 
 .pagination__link--current {
 	font-size: $font-x-small;
-	padding: 15px 15px 0;
+	padding: 15px 15px 0 0;
```

Comment:
OK - I've had a look at most things, it looks good. Just some bits on the profile; but let's merge this 👍